core/src/foundations/ionic.root.scss

```
@@ -12,18 +14,18 @@
   --ionic-elevation-600: 0px 8px 25px 0px rgba(6, 43, 99, 0.08), 0px 1px 5px 0px rgba(6, 43, 99, 0.05);
   --ionic-elevation-700: 0px 15px 32px 0px rgba(6, 43, 99, 0.09), 0px 2px 7px 0px rgba(6, 43, 99, 0.05);
   --ionic-elevation-800: 0px 20px 48px 0px rgba(6, 43, 99, 0.12), 0px 3px 14px 0px rgba(6, 43, 99, 0.12);
-  --ionic-font-size-275: 11px;
-  --ionic-font-size-300: 12px;
-  --ionic-font-size-350: 14px;
-  --ionic-font-size-400: 16px;
-  --ionic-font-size-450: 18px;
-  --ionic-font-size-500: 20px;
-  --ionic-font-size-550: 22px;
-  --ionic-font-size-600: 24px;
-  --ionic-font-size-650: 26px;
-  --ionic-font-size-700: 28px;
-  --ionic-font-size-800: 32px;
-  --ionic-font-size-900: 36px;
+  --ionic-font-size-275: px-to-rem(11);
+  --ionic-font-size-300: px-to-rem(12);
+  --ionic-font-size-350: px-to-rem(14);
+  --ionic-font-size-400: px-to-rem(16);
+  --ionic-font-size-450: px-to-rem(18);
+  --ionic-font-size-500: px-to-rem(20);
+  --ionic-font-size-550: px-to-rem(22);
+  --ionic-font-size-600: px-to-rem(24);
+  --ionic-font-size-650: px-to-rem(26);
+  --ionic-font-size-700: px-to-rem(28);
+  --ionic-font-size-800: px-to-rem(32);
+  --ionic-font-size-900: px-to-rem(36);
```

This function needs to be escaped I believe
```suggestion
  --ionic-font-size-275: #{px-to-rem(11)};
  --ionic-font-size-300: #{px-to-rem(12)};
  --ionic-font-size-350: #{px-to-rem(14)};
  --ionic-font-size-400: #{px-to-rem(16)};
  --ionic-font-size-450: #{px-to-rem(18)};
  --ionic-font-size-500: #{px-to-rem(20)};
  --ionic-font-size-550: #{px-to-rem(22)};
  --ionic-font-size-600: #{px-to-rem(24)};
  --ionic-font-size-650: #{px-to-rem(26)};
  --ionic-font-size-700: #{px-to-rem(28)};
  --ionic-font-size-800: #{px-to-rem(32)};
  --ionic-font-size-900: #{px-to-rem(36)};
```

---

Thats an epic catch, thank you!

---

But will need to adjust the script. Will commit in a bit!